packages/js/eslint.config.mjs

```diff
@@ -83,8 +83,6 @@ export default [
 			"react/jsx-no-bind": "warn",
 			"react/no-access-state-in-setstate": "warn",
 			"react/no-unused-prop-types": "warn",
```

Comment:
Is this still needed?

Reply:
Good shout. Not needed now, `warn` is the same as in the configuration.

Reply:
Not sure how that will work together with the `react/prop-types` and functional components.
We still have `error` in the configuration.
Let's see how this will pan out in the future?

Reply:
[Relevant slack thread](https://yoast.slack.com/archives/C03KU0EHCNQ/p1747641860640199)